test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -85,6 +85,15 @@ class CommunityPermissionsComponents(Enum):
     WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT3 = "communityPermissions_welcome_settings_checkList_element3"
     ADD_NEW_PERMISSION_BUTTON = "communityPermissions_welcome_settings_add_new_permission"
 
+class CommunityTokensComponents(Enum):
```

Comment:
Here I would suggest to remove the `CommunityPermissionsComponent` and the `CommunityTokensComponents` and just add a common generic one for welcome page, since the qml object is the same.  Here my suggestion:

```suggestion
class CommunityWelcomePageComponents(Enum):
    WELCOME_SCREEN_TITLE = "communitySettings_welcome_title"
    WELCOME_SCREEN_IMAGE = "communitySettings_welcome_image"
    WELCOME_SCREEN_SUBTITLE = "communitySettings_welcome_subtitle"
    WELCOME_SCREEN_CHECKLIST_ELEMENT1 = "communitySettings_welcome_checkList_element1"
    WELCOME_SCREEN_CHECKLIST_ELEMENT2 = "communitySettings_welcome_checkList_element2"
    WELCOME_SCREEN_CHECKLIST_ELEMENT3 = "communitySettings_welcome_checkList_element3"
    ADD_NEW_ITEM_BUTTON = "communitySettings_welcome_add_new_permission"
```